addons/io_scene_gltf2/blender/exp/gltf2_blender_generate_extras.py

```diff
@@ -16,6 +16,37 @@
 import bpy
 from io_scene_gltf2.blender.com import gltf2_blender_json
 
+def to_json_compatible(value):
+    """Make a value (usually a custom property) compatible with json"""
+
+    if isinstance(value, bpy.types.ID):
+        return value
+
+    elif isinstance(value, str):
+        return value
+
+    elif isinstance(value, (int, float)):
+        return value
+
+    # for list classes
+    elif isinstance(value, list):
+        value = list(value)
```

Comment:
is this really necessary if it's already of type `list`?

Reply:
It's suppose to clone the list to make sure the original isn't modified. Incompatible custom properties would be set to None in the upcoming loop.

Reply:
okay, that's fine then 👍